sig-node/charter.md

```diff
@@ -0,0 +1,214 @@
+# SIG Node Governance
+
+## Scope
+
+SIG Node is responsible for the components that support the controlled
+interactions between pods and host resources.  We manage the lifecycle of pods
+that are scheduled to a node.  We focus on enabling a broad set of workload
+types, including hardware and performance sensitive workloads.  We maintain
+isolation boundaries between pods on a node, as well as the pod and the host.  We
+aim to continuously improve node reliability.
+
+### In scope
+
+The following topics fall under ownership of this SIG.
+
+* Kubelet related features (e.g. Pod lifecycle)
+* Node level performance and scalability (with [sig-scalability](../sig-scalability))
+* Node reliability (prooblem detection and remediation)
+* Node lifecycle management (with [sig-cluster-lifecycle](../sig-cluster-lifecycle))
+* Container runtimes
+* Device management
+* Images, package management
```

Comment:
I think I know what you mean by package management but it might be confused with SIG release